nix/kaa-client-c/default.nix

```diff
@@ -81,17 +81,17 @@ let
         __propagate:;
       ''
       + target posixSupport "posix"
-              "${lib.optionalString testSupport ''-DCMAKE_BUILD_TYPE=Debug -DKAA_UNITTESTS_COMPILE=on -DKAA_COLLECT_COVERAGE=1''}"
+              "${lib.optionalString testSupport ''-DCMAKE_BUILD_TYPE=Debug -DBUILD_TESTING=on -DKAA_COLLECT_COVERAGE=1''}"
```

Comment:
Doesn't work well for `testSupport = false`. In that case, `BUILD_TESTING` is not set and defaults to `ON`. So this won't disable tests.